pyston/resource.py

```diff
@@ -385,16 +385,16 @@ def _get_response_data(self):
 
         fieldset = True
         try:
+            if self.request.method.lower() not in self.get_allowed_methods():
```

Comment:
I tried following methods: `'get', 'head', 'options', 'put', 'patch', 'delete'` and all of them are recognized here. I just moved the check here, the rest of the logic is left intact.

Reply:
isn't this missing the check for meth?

Reply:
yes you are true, maybe this logic should be moved to get_allowed_methods (return only methods that are implemented)

Reply:
@matllubos I would keep it like it is... IMHO it better separates the allowed methods from no implemented ones (which is a different thing)

Reply:
@Formulka you are right, thanks.